_sections/live-share-code-with-me-support-vscode.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Visual Studio Code
+tab: vscode
+---
+
+#### Starting a Live Share Session 
+If you have the Live Share extension installed on VS Code, you have the option of using Live Share to pair program on your call. To start a Live Share session, you should select the Live Share tab on the Share Code popup modal, and select the project you would like to share. 
+
+![Start Live Share](/uploads/codeshare-start-liveshare.jpg "Start Live Share"){:class="screenshot"}
+
+Once you have selected a project you will be prompted to switch back to your IDE.
+
+![Live Share Open Repo](/uploads/liveshare-open-repo.jpeg "Live Share Open Repo"){:class="screenshot"}
+
+If you have already installed Live Share and you haven't signed in yet, you will be prompted to sign in. 
+
+![Live Share Sign In](/uploads/codeshare-liveshare-sign-in.jpeg "Live Share Sign In"){:class="screenshot"}
+
+
+ Once you have signed in, you are ready for guests to join your session!
+
+#### Joining a Live Share Session 
+
+When your teammate (who will host the code share in this case) starts codesharing via Live Share and you have VS Code open, you will see a prompt to switch to that IDE window.
+
+![Live Share Open Editor](/uploads/codeshare-liveshare-open-editor.jpg "Live Share Open Editor"){:class="screenshot"}
+
+
+Any notifications relating to the codeshare will appear in your IDE window.
+
+![Live Share Join Collab](/uploads/codeshare-liveshare-join-collab.jpeg "Live Share Join Collab"){:class="screenshot"}
+
+
+Your teammate's project will then open in a new IDE window and you are good to go!
+
+If you didn't have a compatible IDE already open you will be prompted to open the sharing link in the browser instead.
```

Comment:
why compatible IDE if this section is vs code live share and only relates to vs code?